clang-tools-extra/test/clang-tidy/checkers/readability/implicit-bool-conversion-check.cpp

```diff
@@ -0,0 +1,86 @@
+// RUN: %check_clang_tidy %s readability-implicit-bool-conversion %t
+
+// RUN: %check_clang_tidy %s readability-implicit-bool-conversion %t \
+// RUN: -- -config='{CheckOptions: [{key: readability-implicit-bool-conversion.CheckConversionsToBool, value: false}, {key: readability-implicit-bool-conversion.CheckConversionsFromBool, value: true}]}'
```

Comment:
this is old format for options, use new one like in other checks

Reply:
Okay Thanks I am Working on it.

Reply:
I have done this . Can you please review it?